dali/operators/ssd/box_encoder.cu

```diff
@@ -94,7 +94,7 @@ __device__ float4 MatchOffsets(
 }
 
 __device__ void WriteMatchesToOutput(
-  int anchors_count, float criteria, int *labels_out, const int *labels_in,
+  unsigned int anchors_count, float criteria, int *labels_out, const int *labels_in,
```

Comment:
Again, just `unsigned`?

Reply:
In this file it's used as `unsigned int` already, so I will keep it.

Reply:
ok